[AGM-2295] Update AdGuard account KB (#621)
AdguardTeam/KnowledgeBase: docs/general/account/2fa.md

## How to set it up

1. Log into your [AdGuard account](https://auth.adguard.com/login.html) and go to the *Settings* tab:

>  ![AdGuard Account settings *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_1.png)

Reviewer: выглядит микро, давай просто border сделаем

Reviewer: (касается всех картинок)

> 1. Scroll down to *Two-factor authentication* section and enable it:

Reviewer: не очень похоже на секцию...
```suggestion
1. Scroll down to *Two-factor authentication* and enable it:
```

>  ![The Change button *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_2.png)

Reviewer: надо переназвать, тут уже нет кнопки

> 1. To enable *Two-factor authentication* a code will be sent to the email address associated with your AdGuard account. Enter the code and click *Enable*.

Reviewer: ```suggestion
1. A code will be sent to the email address associated with your AdGuard account. To enable 2FA, enter the code and click *Enable*.
```

 ![The verification code *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_3.png?)

> After your enabled *Two-factor authentication* there are two options to choose:

Reviewer: ```suggestion
After you enable two-factor authentication, you have two options to choose from:
```

- *One-time email code*: when you log in, we will ask you to enter the code we sent to your email

- *Code from authentication app*: when you log in, we will ask for a code from your authentication app

> ![The verification options *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_4.png)

Reviewer: назвала бы 2FA options или Authentication options

:::note

> Save the second-factor backup code somewhere, it will be necessary to recover your account in case your mobile phone gets lost or stolen.

Reviewer: эта инфа только если юзер захочет использовать authenticator app, нужно это пометить
называется setup key
https://uploads.adguard.org/s.orlova_rej92.png
если будешь использовать мою картинку, перезалей на гугл диск)

:::

If everything is done right, you will see this when you log into your account the next time:

> ![The result *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_success.png)

Reviewer: вот это неактуальная картинка

Reviewer: https://uploads.adguard.org/s.orlova_3lbby.png тоже если будешь использовать, загрузи на гугл диск

## How to set it up

1. Log into your [AdGuard account](https://auth.adguard.com/login.html) and go to the *Settings* tab:

 ![AdGuard Account settings *border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_1.png)

1. Scroll down to *Two-factor authentication* and enable it:

>  ![The toggle button *border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_2.png)

Reviewer: это минорное, но мне кажется так понятнее будет если картинки не видеть
```suggestion
 ![Two-factor authentication toggle *border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_2.png)
```

1. A code will be sent to the email address associated with your AdGuard account. To enable 2FA, enter the code and click *Enable*.

 ![The verification code *border](https://cdn.adtidy.org/content/kb/ad_blocker/general/2fa_3.png?)

After you enable two-factor authentication, you have two options to choose from:

- *One-time email code*: when you log in, we will ask you to enter the code we sent to your email

> - *Code from authentication app*: when you log in, we will ask for a code from your authentication app

Reviewer: кстати по поводу authentication vs authenticator — если погуглить, чаще используется второй вариант
![image](https://github.com/user-attachments/assets/22843fb1-3a5d-4f4d-939e-a77047d97bfd)
предложила бы тут и ниже заменить, но не критично
